app/assets/javascripts/lookup_keys.js

```diff
@@ -300,7 +300,7 @@ function toggle_value_hidden(target, shown){
   if (shown) {
     $(target).replaceWith($('<input/>').attr(_.extend({ type: 'password' }, attrs)));
   } else {
-    $(target).replaceWith($('<textarea/>').attr(_.extend({ placeholder: _('Value'), rows: 1 }, attrs)));
+    $(target).replaceWith($('<textarea/>').attr(_.extend({ placeholder: _('Value'), rows: 1 }, attrs)).val(attrs['value']));
```

Comment:
while you are here, target is already a jQuery object, no need to wrap it in `$()`.